mlops/parallelm/mlops/mlops.py

```diff
@@ -385,18 +388,51 @@ def current_model(self):
         return self._curr_model
 
     @SuppressException([MLOpsConnectionException])
-    def set_stat(self, name, data=None, category=StatCategory.TIME_SERIES, timestamp=None):
+    def set_stat(self, name, data=None, category=StatCategory.TIME_SERIES, timestamp=None, **kwargs):
         """
         Report this statistic.
 
         :param name: name to use in the export
         :param data: data object to export
         :param category: category of the statistic. One of :class:`StatsCategory`
         :param timestamp: optional timestamp
+        :param kwargs: key word arguments is supporting list of arguments for outputing stats correctly. i.e. For outputting Confusion Matrix, user has to provide list of `labels` used.
+        :Example:
+
+        >>> from parallelm.mlops import mlops
+        >>> from parallelm.mlops.metrics_constants import ClassificationMetrics
+
+        >>> # Output Time Series Data
+        >>> mlops.set_stat("TimeSeries", 1.0) # outputing time series data
+
+        >>> # Output ML Stat - For Example Confusion Matrix as Table
+        >>> labels_pred = [1, 0 , 1, 1, 1, 0]
+        >>> labels_actual = [0, 1, 0, 0, 0, 1]
+        >>> labels_ordered = [0, 1]
+
+        >>> ## First Way ##
+        >>> import sklearn
+        >>> from parallelm.mlops.stats.table import Table
+
+        >>> cm = sklearn.metrics.confusion_matrix(labels_actual, labels_pred, labels=labels_ordered)
+        >>> labels_string = [str(i) for i in labels_ordered]
+        >>> cm_matrix = Table().name("User Given Confusion Matrix").cols(labels_string)
+        >>> for index in range(len(cm)):
+        >>>     cm_matrix.add_row(labels_string[index], list(cm[index]))
+        >>> mlops.set_stat(cm_matrix)
+
+        >>> ## OR - Second Way ##
+        >>> mlops.set_stat(ClassificationMetrics.CONFUSION_MATRIX, cm, labels=labels_ordered)
+
+        >>> ## OR - Third Way ##
+        >>> mlops.metrics.confusion_matrix(y_true=labels_actual, y_pred=labels_pred, labels=labels_ordered)
```

Comment:
@lior-amar-pm can you describe why we need this 3rd way of accessing stats? i dont fully understand its need - it feels clunky

Reply:
The third way is syntactic sugar on top of the first way. 
It makes it easier to the user to call this method and to get documentation about the specific params in their development environment (e.g. pycharm, intellij..)

Reply:
@sriram109